shell/platform/android/test/io/flutter/embedding/engine/PluginComponentTest.java

```diff
@@ -32,11 +32,15 @@ public void setUp() {
   @Test
   public void pluginsCanAccessFlutterAssetPaths() {
     // Setup test.
-    FlutterInjector.setInstance(new FlutterInjector.Builder().setShouldLoadNative(false).build());
+    FlutterJNI mockFlutterJNILoader = mock(FlutterJNI.class);
+    FlutterInjector.setInstance(
```

Comment:
Cool, lg. Can you push some changes to https://github.com/flutter/website/pull/4561 directly for the new pattern?

Reply:
https://github.com/flutter/website/pull/4561/commits/301b89ba5bdd0dc41ee79f84e957cd597ee00de9